drivers/base/kdgdb/kdpacket.c

```diff
@@ -379,14 +389,17 @@ KdReceivePacket(
     {
         DBGKD_MANIPULATE_STATE64* State = (DBGKD_MANIPULATE_STATE64*)MessageHeader->Buffer;
 
+        KDDBGPRINT("State manipulation: ");
```

Comment:
Not sure this dprint is interesting, because we always get a corresponding print just afterwards: when the if-block is taken, and when it is not.

Reply:
It's a contextualisation dprint. If you look closely, there is no trailing new line here. It helped me to understand an issue I had, so I keep it there.

Reply:
Out of this PR